### PR TITLE
fix(PDisk): grey color for unknown state

### PR DESCRIPTION
**Pull request by artemmufazalov**

### CODE REVIEW

File: src/containers/Storage/Vdisk/__tests__/colors.tsx [hunk 1]
```diff
@@ -63,26 +63,39 @@ describe('VDisk state', () => {
...
                 <VDisk VDiskId={{Domain: 1}} />
                 <VDisk VDiskId={{Domain: 2}} VDiskState="OK" />
-                <VDisk VDiskId={{Domain: 3}}                 DiskSpace="Green" />
-                <VDisk VDiskId={{Domain: 4}}                                   FrontQueues="Green" />
+                <VDisk VDiskId={{Domain: 3}} DiskSpace="Green" />
+                <VDisk VDiskId={{Domain: 4}} FrontQueues="Green" />
                 <VDisk VDiskId={{Domain: 5}} VDiskState="OK" DiskSpace="Green" />
-                <VDisk VDiskId={{Domain: 6}} VDiskState="OK"                   FrontQueues="Green" />
-                <VDisk VDiskId={{Domain: 7}}                 DiskSpace="Green" FrontQueues="Green" />
-                <VDisk VDiskId={{Domain: 8}} VDiskState="OK" DiskSpace="Green" FrontQueues="Green" />
```
Comment (ariser) [inline src/containers/Storage/Vdisk/__tests__/colors.tsx:71]: those lines were specifically aligned to see the pattern of specified props
it does not match any linter rule, but it was intentional

File: src/containers/Storage/Vdisk/__tests__/colors.tsx [hunk 2]
```diff
@@ -63,26 +63,39 @@ describe('VDisk state', () => {
...
+        // unavailable disks display with the grey color
+        expect(disk1.className).toMatch(/_grey\b/i);
+        expect(disk2.className).not.toMatch(/_grey\b/i);
+        expect(disk3.className).toMatch(/_grey\b/i);
+        expect(disk4.className).toMatch(/_grey\b/i);
+        expect(disk5.className).not.toMatch(/_grey\b/i);
+        expect(disk6.className).not.toMatch(/_grey\b/i);
+        expect(disk7.className).toMatch(/_grey\b/i);
+        expect(disk8.className).not.toMatch(/_grey\b/i);
+        expect(disk9.className).not.toMatch(/_grey\b/i);
```
Comment (ariser) [inline src/containers/Storage/Vdisk/__tests__/colors.tsx:98]: those 8 disks covered all possible combinations of 3 props being defined/undefined
the new 9th doesn't add anything to this test case, this one is not about values, but rather about values presence
if you want to cover coloring in error state, better add another test case

**Review by ariser — CHANGES_REQUESTED** [review]

File: src/containers/Storage/DiskStateProgressBar/DiskStateProgressBar.tsx [hunk 3]
```diff
@@ -53,7 +57,7 @@ export function DiskStateProgressBar({
 
     const mods: Record<string, boolean | undefined> = {inverted};
 
-    const color = severity && severityToColor[severity];
+    const color = severity !== undefined && severityToColor[severity];
```
Comment (artemmufazalov) [inline src/containers/Storage/DiskStateProgressBar/DiskStateProgressBar.tsx:60]: Grey color corresponds to 0 severity, so check `severity &` is not valid here

Comment (ariser) [reply src/containers/Storage/DiskStateProgressBar/DiskStateProgressBar.tsx:60]: good catch, thanks

**Review by ariser — APPROVED** [review]

**Pull request merged**